src/openrct2/CmdlineSprite.cpp

```diff
@@ -46,9 +46,14 @@ assert_struct_size(rct_sprite_file_header, 8);
 
 #pragma pack(pop)
 
-static rct_sprite_file_header spriteFileHeader;
-static rct_g1_element* spriteFileEntries;
-static uint8_t* spriteFileData;
+struct SpriteFile
+{
+    rct_sprite_file_header Header;
+    rct_g1_element* Entries;
+    uint8_t* Data;
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/ade064674ad5327edddbf9f5337eee72d7a5cdd6)

Ìnitialize pointers to `nullptr`

Reply:
Removed in subsequent commits with a std::vector.